### DISCUSSION TITLE
More detailed options for Submission Count for the Plugin Badge

### DISCUSSION THREAD

**Discussion by jrrdnx** [solspace/craft-freeform#279] [Feature Requests]
Currently, if Submission Count is the chosen option for the Plugin Badge, this will display the total number of submissions regardless of status. Since our typical workflow is to leave all submissions in place without deleting them (at least for a time), this count doesn't accurately display the number of unread submissions.

Some clients work solely off of their email notifications, and only use the admin to review form submissions as a fallback. But others will keep new submissions as "Unread" until they can be reviewed, and then mark them as "Complete" or "Read". I know statuses are likely to vary wildly, but it would be nice to limit this badge count to unread submissions only or a select status/statuses.

**Comment by kjmartens** [comment]
Thanks for your feedback. 🙂
